src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/profiledetails/ProfileDetailsSyncService.kt

```diff
@@ -103,8 +103,8 @@ class ProfileDetailsSyncService(
       null
     }
 
-  private fun List<ProfileDetailsResponse>.toDpsRequest(profileType: String) =
-    find { it.type == profileType }
+  private fun ProfileDetailsResponse.toDpsRequestIfType(profileType: String) =
+    takeIf { type == profileType }
```

Comment:
We put a temporary workaround in for DPS concurrent updates problem to send all details at the same time. Now that concurrent updates are fixed we can go back to sending each profile type individually.